src/PrefillGravityForms/Controllers/BaseController.php

```diff
@@ -309,8 +297,21 @@ protected function getCurlHeaders(string $doelBinding = ''): array
         return array_filter($headers);
     }
 
-    protected function handleCurl(array $args): array
+    protected function handleCurl(array $args, string $transientKey): array
     {
+        try {
+            /**
+             * IMPORTANT NOTE: when adjusting this piece of code, please make sure
+             * that the transient key is unique per request. Otherwise, different requests
+             * might return the same cached response.
+             */
+            if ($cachedResponse = CacheService::getArrayFromTransient($transientKey)) {
+                return $cachedResponse;
+            }
+        } catch (Exception $e) {
+            $this->logError('Failed to get transient: ' . $e->getMessage(), $e->getCode());
```

Comment:
Begrijp ik goed dat dit naar een Teams channel logt, zodat de cache niet stuk kan gaan zonder dat wij dat merken? Zelf zou ik zeggen: Als we niet verwachten dat de cache regelmatig stuk zal gaan, catch deze exception dan gewoon niet. Veel eenvoudiger, en dan krijgen we netjes een Sentry error binnen en duiken we erop.